libs/database/src/lib/data/blueprint_page.ts

```diff
@@ -91,6 +107,7 @@ export async function searchBlueprintPages({
       ${itemsFragment}
       ${recipesFragment}
       ${tagsFragment}
+      ${userFragment}
```

Comment:
Please add the userFragment also to the count query below or the pagination/total count will be wrong.